app/routes/route_utilities.py

```diff
@@ -0,0 +1,19 @@
+from flask import abort, make_response
+from ..db import db
+
+
+def validate_model(cls, model_id):
```

Comment:
Great job making this a generic helper function! And great job pulling it out to the route_utilities!